modules/services/redshift-gammastep/lib/options.nix

```diff
@@ -145,21 +197,9 @@ in {
 
       Service = {
         ExecStart = let
-          providerString = if cfg.provider == "manual" then
-            "${cfg.latitude}:${cfg.longitude}"
-          else
-            cfg.provider;
-
-          args = [
-            "-l ${providerString}"
-            "-t ${toString cfg.temperature.day}:${
-              toString cfg.temperature.night
-            }"
-            "-b ${toString cfg.brightness.day}:${toString cfg.brightness.night}"
-          ] ++ cfg.extraOptions;
-
           command = if cfg.tray then appletExecutable else mainExecutable;
-        in "${cfg.package}/bin/${command} ${concatStringsSep " " args}";
+          configFullPath = config.xdg.configHome + "/${xdgConfigFilePath}";
```

Comment:
Why not simply
```suggestion
          configFullPath = settingsFormat.generate xdgConfigFilePath cfg.settings;
```
?

Reply:
This is the official config path for both redshift and gammastep. So if the user wants to call it standalone for any reason, it is possible.

Reply:
I was even thinking of adding something like `serviceEnabled` option so the user could enable/disable the service, but I don't know if it is really worth it.

Reply:
If you want to configure the redshift program without a service then it should go under `programs.redshift`.